src/showcase/assets/stackblitz/src/app/sbb.module.ts

```diff
@@ -0,0 +1,76 @@
+import { NgModule } from '@angular/core';
+import {
```

Comment:
As discussed with bazel rule:
genrule used-icons in src/BUILD.bazel
genrule used-icons in src\angular-core\schematics\BUILD.bazel

Modules in:
- src\angular-business\config.bzl
- src\angular-public\config.bzl

Reply:
I might need more bazel knowledge to do this, I seem to get stuck on very basic problems e.g. how to test my changed bazel BUILD files... if you want I can invest some time into learning more about bazel, otherwise I'll leave this one up to you ;)

Reply:
Maybe it would be easier to create two separate files for public and business and adapt the import path in app.module.ts depending on public or business.

I just checked how to  pass variables to a genrule and it is not simple: https://docs.bazel.build/versions/2.0.0/be/make-variables.html#custom_variables

If you want, you can investigate how to do this in bazel, but you don't have to.